packages/aws-appsync/src/link/signer/signer.js

```diff
@@ -217,7 +217,7 @@ var sign = function (request, access_info, service_info = null) {
 
     // datetime string and date string
     var dt = new Date(),
-        dt_str = dt.toISOString().replace(/[:\-]|\.\d{3}/g, ''),
+        dt_str = dt.toISOString().replace(/[:-]|\.\d{3}/g, ''),
```

Comment:
where is ```dt_str``` defined?

Reply:
It is part of the same [`var` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#Syntax)

Reply:
lol thanks!